public/config/configuration.json

```diff
@@ -4,5 +4,7 @@
   "STATUS_API_URL": "https://status-api-dev.apps.silver.devops.gov.bc.ca",
   "STATUS_API_VERSION": "/api/v1",
   "PAY_API_URL": "https://pay-api-dev.apps.silver.devops.gov.bc.ca",
-  "PAY_API_VERSION": "/api/v1"
+  "PAY_API_VERSION": "/api/v1",
+  "AUTH_WEB_URL": "https://dev.bcregistry.ca/business/auth/",
```

Comment:
do we need auth-web here? better to pass from there

Reply:
We would need it for refresh issue and that meta is not a good option, we would need for now. Once we move it to a session storage in the next ticket, we can remove here